src/main/java/com/spikes2212/command/genericsubsystem/MotoredGenericSubsystem.java

```diff
@@ -15,48 +15,80 @@
 public class MotoredGenericSubsystem extends GenericSubsystem {
 
     protected MotorControllerGroup motorControllerGroup;
-    protected RootNamespace rootNamespace;
 
     /**
-     * Constructs a new instance of MotoredGenericSubsystem with the given {@code RootNamespace}'s name, the given
-     * maxSpeed supplier, the given minSpeed supplier and the given {@code MotorController}s.
+     * Constructs a new instance of {@link MotoredGenericSubsystem} with the given {@link RootNamespace}'s name, the given
+     * minSpeed supplier, the given maxSpeed supplier and the given {@link MotorController}s.
      *
+     * @param namespaceName    the name of the subsystem's namespace
      * @param minSpeed         the minimum speed
      * @param maxSpeed         the maximum speed
-     * @param namespaceName    the name of the subsystem's namespace
      * @param motorControllers the motor controllers in the subsystem
      */
-    public MotoredGenericSubsystem(Supplier<Double> minSpeed, Supplier<Double> maxSpeed, String namespaceName,
+    public MotoredGenericSubsystem(String namespaceName, Supplier<Double> minSpeed, Supplier<Double> maxSpeed,
                                    MotorController... motorControllers) {
-        super(minSpeed, maxSpeed);
+        super(namespaceName, minSpeed, maxSpeed);
         this.rootNamespace = new RootNamespace(namespaceName);
         this.motorControllerGroup = new MotorControllerGroup(motorControllers);
     }
 
     /**
-     * Constructs a new instance of MotoredGenericSubsystem with the given {@code RootNamespace}'s name, the given
-     * maxSpeed, the given minSpeed and the given {@code MotorController}s.
+     * <p>Constructs a new instance of {@link MotoredGenericSubsystem} with the given
```

Comment:
this is definitely not how paragraph tags should be used

Reply:
the paragraph tag as a concept is dumb
there is not really a good way to use it